install/kubernetes/helm/istio/charts/gateways/templates/_affinity.tpl

```diff
@@ -41,3 +41,50 @@
     {{- end }}
   {{- end }}
 {{- end }}
+
+{{- define "gatewaypodAntiAffinity" }}
+{{- if or .podAntiAffinityLabelSelector .podAffinityTermLabelSelector}}
+  podAntiAffinity:
+    {{- if .podAntiAffinityLabelSelector }}
+    requiredDuringSchedulingIgnoredDuringExecution:
+    {{- include "gatewaypodAntiAffinityRequiredDuringScheduling" . }}
+    {{- end }}
+    {{- if or .podAffinityTermLabelSelector}}
+    preferredDuringSchedulingIgnoredDuringExecution:
+    {{- include "gatewaupodAntiAffinityPreferredDuringScheduling" . }}
```

Comment:
typo, should be `gatewaypodAntiAffinityPreferredDuringScheduling `

Reply:
thanks! done